joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 100

door_opening_joint_velocity_controller:
    type: moma_gazebo/DoorOpeningJointVelocityController
    arm_id: panda
    joint_names:
        - panda_joint1
        - panda_joint2
        - panda_joint3
        - panda_joint4
        - panda_joint5
        - panda_joint6
        - panda_joint7
    max_duration_between_commands: 0.1
    max_velocity: [2.1750, 2.1750, 2.1750, 2.1750, 2.6100, 2.6100, 2.6100]
    max_acceleration: [15, 7.5, 10, 12.5, 15, 20, 20]
    max_jerk: [7500, 3750, 5000, 6250, 7500, 10000, 10000]

franka_gripper:
  type: position_controllers/GripperActionController
  joint: panda_finger_joint1

gazebo_ros_control:
  pid_gains:
    panda_finger_joint1: {p: 100, i: 0.1, d: 10, i_clamp: 0.2}
    panda_finger_joint2: {p: 100, i: 0.1, d: 10, i_clamp: 0.2}
    front_left_wheel: {p: 1, i: 0.1, d: 0, i_clamp: 0.2}
    front_right_wheel: {p: 1, i: 0.1, d: 0, i_clamp: 0.2}
    rear_left_wheel: {p: 1, i: 0.1, d: 0, i_clamp: 0.2}
    rear_right_wheel: {p: 1, i: 0.1, d: 0, i_clamp: 0.2}

